### DISCUSSION TITLE
plugin [Jul 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2845] [Discord Threads]
> 🙋‍♂️ **moi**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Which plugin you had in mind. If it's for a category that already exists  
#  
> 💁 **o1lab**  
#### You can extend it  
#  
> 💁 **o1lab**  
#### Else will have to be created  
#  
> 🙋‍♂️ **moi**  
#### I'd like to use nocodb for some administration workers. They have some specifics needs and I'd like to extend nocodb to fit some specifics needs.   
#### I thought the plugins system would be perfect but as I dig in the code, it seems the plugins are stored inside nocodb code base.  
#  
> 🙋‍♂️ **moi**  
#### Do you think it's a good solution ?  
#  
> 💁 **o1lab**  
#### we would like to ideally keep it outside   
#### however, would liek to know more on the usevase  
#  
> 🙋‍♂️ **moi**  
#### Keep it outside what ? The plugin system ?  
#  
> 🙋‍♂️ **moi**  
#### How can I extend nocodb features ? Do you have some documentation about it ?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/997164606881529927)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
